src/main/java/org/opensearch/security/configuration/ConfigurationRepository.java

```diff
@@ -145,6 +145,7 @@ public void run() {
                                     final boolean populateEmptyIfFileMissing = true;
                                     ConfigHelper.uploadFile(client, cd+"nodes_dn.yml", securityIndex, CType.NODESDN, DEFAULT_CONFIG_VERSION, populateEmptyIfFileMissing);
                                     ConfigHelper.uploadFile(client, cd + "whitelist.yml", securityIndex, CType.WHITELIST, DEFAULT_CONFIG_VERSION, populateEmptyIfFileMissing);
```

Comment:
should we still keep this?

Reply:
Yes, whitelist will be supported(deprecated) till 3.0.0.